core/main/src/state/metrics_state.rs

```diff
@@ -56,6 +76,19 @@ impl MetricsState {
         *cache = value.clone();
     }
     pub async fn initialize(state: &PlatformState) {
+        let metrics_percentage = state
+            .get_device_manifest()
+            .configuration
+            .metrics_logging_percentage;
+
+        let random_number = rand::thread_rng().gen_range(1..101);
+        let metrics_enabled = random_number <= metrics_percentage;
+
+        debug!(
+            "initialize: metrics_percentage={}, random_number={}, enabled={}",
+            metrics_percentage, random_number, metrics_enabled
+        );
+
```

Comment:
En/disables metrics collection based on percentage threshold in manifest.

Reply:
Guessing QA can just set 100 on the manifest to make it enabled